### DISCUSSION TITLE
14.1.0

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#4735] [General]
## Summary

This release adds a Specials folder for TV series and is packed with massive performance improvements to scanning and browsing, as well as fixes for many bugs you reported to us.

Thank you so much for the community engagement we are seeing, it really helps us continue to build in the right direction for you.

This release is available via [our Patreon page](https://www.patreon.com/posts/14-1-0-105663716), and will be available here when the next release happens.

### General
- Added Specials folder for TV Series
- Jetty is now a selectable media server engine, with JUPnP+
- The web interface / web player are now served by Jetty
- Improved scanning and browsing speed
- Improved browsing responsiveness during scans
- Implemented CDS UpdateObject base functionality for audio
- Fixed TV episodes sorting (#4694) (thanks, @MEGAKNIGHT88!)
- Fixed sorting by date (#4698) (thanks, @mykeehu!)
- Improved web interface build speed with Vite
- General bugfixes
- Removed the JDK HTTP servlet server personal implementation

### Media players
- Fixed support for media players (like some Samsung TVs) that rely on X_GetFeatureList (#4713) (thanks, @si70so15!)

### Translation updates via Crowdin
- Japanese (52%) (thanks, mict213!)
- Portuguese (62%) (thanks, paulo santos!)

### Dependencies
- Bump `@mantine/core` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@mantine/dropzone` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@mantine/form` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@mantine/hooks` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@mantine/modals` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@mantine/notifications` from 7.10.0 to 7.10.1 ([#4718](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4718))
- Bump `@types/lodash` from 4.17.1 to 4.17.4 ([#4706](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4706))
- Bump `@types/react` from 18.3.1 to 18.3.3 ([#4720](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4720))
- Bump `@typescript-eslint/eslint-plugin` from 7.9.0 to 7.11.0 ([#4705](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4705), [#4719](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4719))
- Bump `@typescript-eslint/parser` from 7.9.0 to 7.11.0 ([#4705](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4705), [#4719](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4719))
- Bump `axios` from 1.6.8 to 1.7.2 ([#4721](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4721))
- Bump `com.github.oshi:oshi-core` from 6.6.0 to 6.6.1 ([#4710](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4710))
- Bump `com.ibm.icu:icu4j` from 74.2 to 75.1 ([#4673](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4673))
- Bump `com.puppycrawl.tools:checkstyle` from 10.12.7 to 10.17.0 ([#4723](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4723))
- Bump `com.sun.xml.bind:jaxb-impl` from 4.0.4 to 4.0.5 ([#4711](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4711))
- Bump `commons-logging:commons-logging` from 1.2 to 1.3.2 ([#4726](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4726))
- Bump `io.github.git-commit-id:git-commit-id-maven-plugin` from 7.0.0 to 8.0.2 ([#4624](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4624))
- Bump `jetty.version` from 12.0.8 to 12.0.9 ([#4724](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4724))
- Bump `mantine` from 7.5.3 to 7.10.0 ([#4704](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4704))
- Bump `Node.js` from 18.14.0 to 20.12.2
- Bump `org.codehaus.mojo:exec-maven-plugin` from 3.2.0 to 3.3.0 ([#4709](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4709))
- Bump `org.digitalmediaserver:nsis-maven-plugin` from 1.0.5 to 1.0.6 ([#4675](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4675))
- Bump `org.jupnp:org.jupnp.support` from 2.7.1 to 3.0.2 ([#4672](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4672))
- Bump `tmdbapi` from 0.2 to 0.3 ([#4663](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/4663))
- Bump `video.js` from 8.12.0 to 8.14.0
- Bump `yarn` from 4.1.1 to 4.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/14.1.0'>14.1.0</a>.</em>

**Comment by MEGAKNIGHT88** [comment]
Great!

**Reply by MEGAKNIGHT88** [reply]
It's very fabulous to see how the program is evolved with a time! There are still quite few discontinued settings and more prospects for improvement, for ex. , it's been started after some version, the VLC engine stopped working at all, couldn't play anything... 
P.s. Leave a message on a forum to PM sometime ago.

**Comment by MEGAKNIGHT88** [comment]
When it will be released, approximately? Waiting so long....

**Reply by SubJunk** [reply]
I'm building the 14.2.0 release now :)

**Reply by MEGAKNIGHT88** [reply]
Tnx, the latest release works very well, nevertheless of issues that could be!
